doc/source/data/examples/index.rst

```diff
@@ -0,0 +1,51 @@
+.. _datasets-examples-ref:
+
+========
+Examples
+========
+
+.. tip:: Check out the Datasets :`User Guide <data_user_guide>` To learn more about
+  Datasets' features in-depth.
+
+.. _datasets-recipes:
+
+Basic Data Processing Examples
+------------------------------
+
+Ray Datasets is a powerful engine for data processing that supports multiple data
```

Comment:
This group of guides will be focused on demonstrating both support for different data modalities AND data processing functionality, so I'd like to call out data processing here, if possible. Any particular reason you don't like the call out?

Reply:
I just think it doesn't add much with "powerful (or great) engine" here. It may be more common to say a bit alternatively, like: Ray Datasets is a data processing library capable of multiple data modalities... and then showcase that in our examples.